docs/v3/develop/settings-and-profiles.mdx

```diff
@@ -260,6 +260,10 @@ Self-hosted Prefect servers can be equipped with Basic Authentication through tw
 
 With these settings, the UI will prompt for the full authentication string `"admin:pass"` (no quotes) upon first load. It is recommended to store this information in a secure way, such as a Kubernetes Secret or in a private `.env` file.
 
+If both Prefect API key and Prefect API authentication string are set on the client, Prefect API key will be used. If you plan to use a
```

Comment:
```suggestion
If both `PREFECT_API_KEY` and `PREFECT_API_AUTH_STRING` are set on the client, `PREFECT_API_KEY` will take precedence. If you plan to use a
```

Reply:
that's true! i wouldn't say it's wrong as is, just the vast majority use env vars